docs/framework/guides/package-generator.md

```diff
@@ -176,6 +183,54 @@ Examples:
 npm run dll:serve
 ```
 
+### `translations:collect`
+
+Collects translation messages (arguments of the `t()` function) and context files, then validate whether provided values do not interfere with the values specified in the `@ckeditor/ckeditor5-core` package.
+
+The task may end with an error if one of the following conditions is met:
+
+* Found the "Unused context" error &ndash; entries specified in the `lang/contexts.json` file are not used in source files. They should be removed.
+* Found the "Context is duplicated for the id" error &ndash; some of the entries are duplicated. Consider removing them from the `lang/contexts.json` file, or rewrite them.
+* Found the "Context for the message id is missing" error &ndash; entries specified in source files are not described in the `lang/contexts.json` file. They should be added.
```

Comment:
```suggestion
* Found the `Unused context` error &ndash; entries specified in the `lang/contexts.json` file are not used in source files. They should be removed.
* Found the `Context is duplicated for the id` error &ndash; some of the entries are duplicated. Consider removing them from the `lang/contexts.json` file, or rewrite them.
* Found the `Context for the message id is missing` error &ndash; entries specified in source files are not described in the `lang/contexts.json` file. They should be added.
```

Perhaps error messages could be highlighted as a code? Alternatively, they could be in **bold** or _italic_.